auto_dev/data/repo/templates/autonomy/Makefile.template

```diff
@@ -56,8 +58,25 @@ test:
 	poetry run adev -v test
 
 install:
+	@echo "Setting up Git hooks..."
+	# Ensure the hooks directory exists
+	@if [ ! -d "$(HOOKS_DIR)" ]; then \
+		echo "Error: Git hooks directory does not exist. Initialize a Git repository first."; \
+		exit 1; \
+	fi
+	# Create symlinks for pre-commit and pre-push hooks
+	ln -sf ../../scripts/pre_commit_hook.sh $(HOOKS_DIR)/pre-commit
```

Comment:
Symbolic links do not work on windows as far as I know just copy the files

Reply:
#367